vendor/bat-native-ledger/src/bat/ledger/internal/promotion/promotion.cc

```diff
@@ -193,6 +193,13 @@ void Promotion::OnGetAllPromotions(
     return;
   }
 
+  // even though that some promotions are corrupted
+  // we should display non corrupted ones either way
+  if (success == ledger::Result::CORRUPTED_DATA) {
+    BLOG(ledger_, ledger::LogLevel::LOG_ERROR) <<
+        "Some promotions are not formatted correctly";
```

Comment:
Could we not log the ids of the corrupted promotions or anything that would help us identify the issue?

Reply:
added